lib/logging/owner_unix.go

```diff
@@ -0,0 +1,28 @@
+// +build !windows
+package logging
```

Comment:
It needs a new line between build constraint and package.
refer to [how-to-use-conditional-compilation-with-the-go-build-too](https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool)